integration-tests/tests/snap_op_test.go

```diff
@@ -49,3 +55,24 @@ func (s *snapOpSuite) testInstallRemove(c *check.C, snapName, displayName string
 func (s *snapOpSuite) TestInstallRemoveAliasWorks(c *check.C) {
 	s.testInstallRemove(c, "hello-world", "hello-world")
 }
+
+func (s *snapOpSuite) TestRemoveRemovesAllRevisions(c *check.C) {
+	snapPath, err := build.LocalSnap(c, data.BasicSnapName)
+	defer os.Remove(snapPath)
+	c.Assert(err, check.IsNil, check.Commentf("Error building local snap: %s", err))
+
+	// install two revisions
+	installSnap(c, snapPath)
+	installOutput := installSnap(c, snapPath)
+	c.Assert(installOutput, testutil.Contains, data.BasicSnapName)
+	// double check, sideloaded snaps have revnos like 1000xx
+	revnos, _ := filepath.Glob(filepath.Join(dirs.SnapSnapsDir, data.BasicSnapName, "1*"))
+	c.Check(len(revnos) >= 2, check.Equals, true)
```

Comment:
Shouldn't we stop if we don't have at least 2 revnos (ie. use Assert instead of Check)?